_posts/2023-11-01-sr2024-kickstart-and-rules.md

```diff
@@ -2,13 +2,15 @@
 title: SR2024 Kickstart Out Of This World
 ---
 
-Student Robotics 2024 was Kickstarted on Saturday 21st October on our livestream. 30 teams from around the UK and Europe joined us as we introduced competitors to the competition, our Kits, and this year’s game! Teams will have six months to build, program, and test a fully autonomous robot, learning invaluable technical and soft skills along the way.
```

Comment:
Ah, I had been thinking to keep this sentence (in case people don't read too far, plus it sets up the general context well). I think it's ok that this duplicates a little of what we say later.